sdk/communication/Azure.Communication.Calling.Server/tests/ServerCallingClients/CallingServerClientsLiveTests.cs

```diff
@@ -0,0 +1,243 @@
+        [Ignore("PlayAudio Operation required the call is under established state which is not doable for now.")]
+        public async Task PlayAudioTest()
+        {
+            CallClient client = CreateInstrumentedCallingServerClient();
+            try
+            {
+                var createCallResponse = await CreateCallOperation(client).ConfigureAwait(false);
+                var callLegId = createCallResponse.Value.CallLegId;
+
+                await PlayAudioOperation(client, callLegId).ConfigureAwait(false);
+                await DeteleCallOperation(client, callLegId).ConfigureAwait(false);
+            }
+            catch (RequestFailedException ex)
+            {
+                Console.WriteLine(ex.Message);
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+        }
+
+        [Test]
+        [Ignore("Ignore for now as we get build errors that block checkingin.")]
+        public async Task HangupCallTest()
+        {
+            CallClient client = CreateInstrumentedCallingServerClient();
+            try
+            {
+                var createCallResponse = await CreateCallOperation(client).ConfigureAwait(false);
+                var callLegId = createCallResponse.Value.CallLegId;
+
+                await HangupOperation(client, callLegId).ConfigureAwait(false);
+                await DeteleCallOperation(client, callLegId).ConfigureAwait(false);
+            }
+            catch (RequestFailedException ex)
+            {
+                Console.WriteLine(ex.Message);
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+        }
+
+        [Test]
+        [Ignore("CancelMediaOperations Operation required the call is under established state which is not doable for now.")]
+        public async Task CancelMediaOperationsTest()
+        {
+            CallClient client = CreateInstrumentedCallingServerClient();
+            try
+            {
+                var createCallResponse = await CreateCallOperation(client).ConfigureAwait(false);
+                var callLegId = createCallResponse.Value.CallLegId;
+
+                await CancelMediaOperationsOperation(client, callLegId).ConfigureAwait(false);
+                await DeteleCallOperation(client, callLegId).ConfigureAwait(false);
+            }
+            catch (RequestFailedException ex)
+            {
+                Console.WriteLine(ex.Message);
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+        }
+
+        #region Support functions
+        #region Snippet:Azure_Communication_ServerCalling_Tests_CreateCallOperation
```

Comment:
We usually define these regions to be used on our README files. I dont see this being referred there, what is the reason for defining this region?

Reply:
I see quite a few examples of this, wondering why we have them

Reply:
We will add the Snippet: of those scenarios as samples later.